pkg/prowgen/generators.go

```diff
@@ -151,7 +151,7 @@ func E2ETest(ctx *ProwContext, k8sVersion string) *Job {
 		addMaxConcurrency(4),
 	)
 
-	makeJobs, cpuRequest := calculateMakeConcurrency("3500m")
+	makeJobs, cpuRequest := calculateMakeConcurrency("7000m")
```

Comment:
FWIW this value is not used for the upgrade test, so there is a diff with the current config in testing where this was manually changed to `700m` https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/cert-manager/master/cert-manager-master.yaml#L980
If the additional CPU was needed because of the e2e test setup changes where we use more concurrent workers and higher burst/QPS for kube queries, then that's ok as the upgrade test installs cert-manager using a separate install command that doesn't have those opts

Reply:
Indeed, the upgrade command does not need 7000m. Also, there might be more tuning necessary in the future.